xml_converter/intigration_tests/testcases.py

```diff
@@ -119,6 +119,11 @@ class Testcase:
         expected_output_xml_path="./expected_outputs/xml_profession_filter",
         expected_output_proto_path="./expected_outputs/proto_profession_filter",
     ),
+    Testcase(
+        name="animation_speed",
+        xml_input_paths=["./inputs/xml_animation_speed"],
+        expected_output_xml_path="./expected_outputs/xml_animation_speed",
+    ),
```

Comment:
Where are the proto testcases?

Reply:
Why not just change the proto here? Now seems like as good of a time as any to avoid it being forgotten.